Packs/Armis/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Armis",
     "description": "Agentless and passive security platform that sees, identifies, and classifies every device, tracks behavior, identifies threats, and takes action automatically to protect critical information and systems",
     "support": "partner",
-    "currentVersion": "1.1.20",
+    "currentVersion": "1.2.0",
```

```suggestion
    "currentVersion": "1.1.21",
```
Is this really a minor update and not a revision?
For more info [see](https://xsoar.pan.dev/docs/documentation/release-notes)
Change the release notes filename accordingly